CMakeLists.txt

```diff
@@ -46,7 +46,7 @@
 #
 # ============================================================================
 
-cmake_minimum_required (VERSION 2.8)
```

Comment:
By setting the minimum required version the policy is implicitly set, so it is not needed in the pull request.